GitCommands/Git/CommandCache.cs

```diff
@@ -63,10 +64,10 @@ public IReadOnlyList<string> GetCachedCommands()
         /// <param name="output">Stored output bytes of the command, if found.</param>
         /// <param name="error">Stored error bytes of the command, if found.</param>
         /// <returns><c>true</c> if the command was found, otherwise <c>false</c>.</returns>
-        public bool TryGet(string cmd, [NotNullWhen(returnValue: true)] out byte[]? output, [NotNullWhen(returnValue: true)] out byte[]? error)
+        public bool TryGet(string? cmd, [NotNullWhen(returnValue: true)] out byte[]? output, [NotNullWhen(returnValue: true)] out byte[]? error)
         {
             // Never cache empty commands
-            if (!string.IsNullOrEmpty(cmd))
+            if (!Strings.IsNullOrEmpty(cmd))
```

Comment:
Should have realized that this needed to be fixed before pushing...

Reply:
I'm not sure what you mean @gerhardol.

Reply:
I did not fix the annotations before pushing.